functions/pipeline/shared/db_access_v2.py

```diff
@@ -75,7 +77,8 @@ def get_new_images(self, number_of_images, user_id):
         try:
             conn = self._db_provider.get_connection()
             try:
-                cursor = conn.cursor()       
+                cursor = conn.cursor()
+                # TODO: Should we add TagStateId = INCOMPLETE_TAG also for fetching images?
```

Comment:
thoughts on adding priority for INCOMPLETE_TAG? Should we rank checkout priority?